about.css

```diff
@@ -0,0 +1,647 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --header-shadow-color: #00000040;
+  --primary-color: #272a31;
+  --secondary-color: #ec5242;
+  --light-gray-color: #d3d3d3;
+  --nav-item-color: #868686;
+  --about-home-color: #6d3b3a;
+  --hamburger-menu-color: #3e3e3e;
+  --black-color: #000;
+  --gray-color: #f8f8f8;
+  --para-two-color: #515151;
+  --main-font: "Lato", sans-serif;
+  --font-cocogoose: "Signika Negative", "san-serif";
+}
+
+header {
+  box-shadow: 4px 4px 4px 0 var(--header-shadow-color);
+}
+
+.nav-branding img {
+  width: 190px;
+  height: 80px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.navbar {
+  min-height: 75px;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 0 24px;
+}
+
+.nav-menu {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+}
+
+.nav-link {
+  font-family: var(--main-font);
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 17px;
+  letter-spacing: 0;
+  text-align: left;
+  color: var(--nav-item-color);
+  transition: 0.3s ease;
+}
+
+.nav-link:hover {
+  color: var(--black-color);
+  background: var(--secondary-color);
+  padding: 30px;
+}
+
+.orange-text {
+  color: var(--secondary-color);
+}
+
+.hamburger {
+  display: none;
+}
+
+.hero {
+  background-image: url();
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  height: 100vh;
+  padding-top: 20%;
+  text-align: center;
+}
+
+.quote {
+  font-family: var(--main-font);
+  color: var(--secondary-color);
+  margin-bottom: 10px;
+}
+
+.first-heading {
+  font-family: var(--font-cocogoose);
+  font-size: 1.8rem;
+  color: transparent;
+  background-image: url(./images/download.png);
+  background-size: cover;
+  -webkit-background-clip: text;
+  font-weight: 900;
+  text-transform: uppercase;
+}
+
+.first-para {
+  margin: 10px 15px;
+  background: #fff;
+  padding: 10px;
+}
+
+.first-para p {
+  font-family: var(--main-font);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 17px;
+  letter-spacing: -0.01em;
+  color: var(--nav-item-color);
+}
+
+.second-heading {
+  font-family: var(--main-font);
+  font-size: 16px;
+  font-weight: 900;
+  letter-spacing: 0.0001rem;
+  color: var(--para-two-color);
+  margin: 67px 10px 10px 5px;
+}
+
+.second-para {
+  font-family: var(--main-font);
+  font-size: 14px;
+  font-weight: 400;
+  letter-spacing: 0.0001rem;
+  margin-top: 5px;
+}
+
+.second-para a {
+  color: #595959;
+  text-decoration: underline;
+}
+
```

Comment:
Please  kindly remove grey background-color from .about-first-section as it doesn't look good and is not part of the style design guideline given in behance , it should be a white background with a background image if possible

Behance design :
![image](https://github.com/peter-mwovi/Capstone-project/assets/20537712/4730a84c-dcd3-4de5-a723-b5d8773cba30)

Your design :
![image](https://github.com/peter-mwovi/Capstone-project/assets/20537712/089734da-443b-43ef-9e05-705fc2ba8521)

Reply:
Resolved.